doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc

```diff
@@ -9,100 +9,92 @@ You can use the process instance migration (PIM) service to create, export and e
-. Unzip the downloaded archive.
-. Move the `{PRODUCT_FILE}-process-migration-service-standalone.jar` file from the add-ons archive to a desired location.
-. Start the PIM service:
+. Extract the `{PRODUCT_FILE}-add-ons.zip` file.
+
+. Extract the `{PRODUCT_FILE}-process-migration-service.zip` file.
+
+. Create the database schema:
+
+.. Download the schema creation files:
 +
-[source,subs="attributes+"]
+[source]
 ----
-$ java -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
+$ curl https://raw.githubusercontent.com/kiegroup/process-migration-service/main/ddl-scripts/postgresql/postgresql-quartz-schema.sql -o postgresql-quartz-schema.sql
+$ curl https://raw.githubusercontent.com/kiegroup/process-migration-service/main/ddl-scripts/postgresql/postgresql-pim-schema.sql -o postgresql-pim-schema.sql
 ----
-. Ensure the relevant authentication method is set for `auth-method`. Valid values are `file`, `jdbc`, and `ldap`. For example:
+
+.. Create the tables:
 +
-.`ldap` method example
+[source]
 ----
-pim:
-  auth-method: ldap
-quakus:
-  security:
-    ldap:
-      dir-context:
-        url: ldap://localhost:10389
-        principal: uid=admin,ou=system
-        password: secret
-      identity-mapping:
-        search-base-dn: ou=users,dc=example,dc=com
-        rdn-identifier: uid
-        attribute-mappings:
-          from: uid
-          filter: uniqueMember
-          filter-base-dn: ou=groups,dc=example,dc=com
+$ psql -U jboss -h localhost -d rhpam7 -f ~/Downloads/postgresql-quartz-schema.sql
+$ psql -U jboss -h localhost -d rhpam7 -f ~/Downloads/postgresql-pim-schema.sql
 ----
+
+. Change directory to the `process-migration` directory.
+
+. Use a text editor to create the `servers.yaml` configuration file with the following content and save in the `process-migration` directory:
 +
-.`file` method example
-+
+[source]
 ----
-pim:
-  auth-method: file
-quarkus:
-  security:
-    users:
-      file:
-        enabled: true
-        plain-text: true
-        users: users.properties
-        roles: roles.properties
+kieservers:
+  - host: http://localhost:8080/kie-server/services/rest/server
+    username: rhpamAdmin
+    password: password1!
 ----
+
+. Use a text editor to create the `datasource.yaml` configuration file with the following content `datasource.yaml` and save in the `process-migration` directory:
 +
-.`jdbc` method example
-+
+[source]
 ----
-pim:
-  auth-method: jdbc
 quarkus:
-  jdbc:
-    principal-query:
-      sql: SELECT u.password, u.role FROM users u WHERE u.username=?
-      clear-password-mapper:
-        enabled: true
-        password-index: 1
-      attribute-mappings:
-        0:
-          index: 2
-          to: groups
+  datasource:
+    db-kind: postgresql
+    jdbc:
+      url: jdbc:postgresql://localhost:5432/rhpam7
+    username: jboss
+    password: jboss
```

Comment:
Same here, or add a comment that's just an example that may be tuned to the corresponding configuration for the user

Reply:
Updated.